genologics/entities.py

```diff
@@ -499,8 +503,8 @@ def __setitem__(self, key, value):
                                           nsmap('udf:field'),
                                           type=vtype,
                                           name=key)
-            if not isinstance(value, unicode):
-                value = unicode(str(value), 'UTF-8')
+            if not isinstance(value, str):
+                value = str(str(value), 'UTF-8')
```

Comment:
while this is correct in Python 3, str does not take more than one arg in python2.